app/assets/templates/gauge_template.html.erb

```diff
@@ -0,0 +1,107 @@
+        <div ng-show="showControls || showTab" class="btn btn-primary btn-sm" ng-click="toggleTab($event, 'datasources')" tooltip-append-to-body="true" tooltip="Datasources"><i class="icon-add-to-list"></i></div>
+        <div ng-show="showControls || showTab" class="btn btn-primary btn-sm" ng-click="toggleTab($event, 'graph')" tooltip-append-to-body="true" tooltip="Chart settings"><i class="icon-pie-chart"></i></div>
+        <div ng-show="showControls || showTab" class="btn btn-primary btn-sm" ng-click="toggleTab($event, 'palette')" tooltip-append-to-body="true" tooltip="Palette"><i class="icon-palette"></i></div>
+        <div ng-show="(showControls || showTab) && !widgetPage" class="btn btn-primary btn-sm" ng-click="toggleTab($event, 'staticlink'); generateWidgetLink($event)" tooltip-append-to-body="true" tooltip="Link to Chart"><i class="icon-link"></i></div>
+        <div ng-show="showControls || showTab" class="btn btn-primary btn-sm" ng-click="widgetDeleteModal()" tooltip-append-to-body="true" tooltip="Remove chart"><i class="icon-trash"></i></div>
+      </div>
+
+    </div>
+
+    <div class="pull-left graph_control_tabbar" ng-show="showTab">
+      <!-- Chart control tab content panes -->
+      <!-- 1) Datasources -->
+      <div class="graph_control_tabpane panel panel-default" ng-show="showTab == 'datasources'">
+        <div class="panel-heading">
+          Datasources
+          <i ng-click="closeTab($event)" class="clone_widget_close pointer pull-right icon-cross"
+           tooltip-append-to-body="true" tooltip="Close"></i>
+        </div>
+        <div class="panel-body">
+          <expression
+            axes="graph.axes"
+            end-time="graph.endTime"
+            expr="graph.expression"
+            range="graph.range"
+            servers="servers"
+            servers-by-id="serversById"
+            type="{{graph.type}}"
+            vars="vars"
+            index="$index"></expression>
+        </div>
+      </div>
+
+      <!-- 2) Chart Settings -->
+      <div class="graph_control_tabpane panel panel-default" ng-show="showTab == 'graph'">
+        <div class="panel-heading">
+          Chart Settings
+          <i ng-click="closeTab($event)" class="clone_widget_close pointer pull-right icon-cross"
+           tooltip-append-to-body="true" tooltip="Close"></i>
+        </div>
+        <div class="panel-body">
+          <label>Gauge Title:</label>
+          <input class="form-control" ng-model="graph.title">
+          <label>Gauge Max:</label>
+          <input class="form-control" type="number" ng-model="graph.max">
+          <label>Gauge Units:</label>
+          <input class="form-control" ng-model="graph.units">
+          <label>Refresh interval (seconds):</label>
```

Comment:
Let's just go with `1m` unless you have the global refresh rate readily available. But maybe the latter doesn't make sense anyways, since we probably also don't want to update the widget's refresh rate when changing the global refresh.